### ISSUE TITLE
DOOM II in Woof source port doesn't have my saves

### ISSUE THREAD

**Issue by amethystgosling163** [Youda008/DoomRunner#194]
Added the woof appimage to the engines. Set config directory as `/home/danish/.local/share/woof` and data directory as `/home/danish/.local/share/woof/savegames`. Then added my IWADs folder, and finally set `~/.local/share/woof/woof.cfg` as my config.

Yet, when I launch my DOOM II through the launcher, it doesn't have any of my saves. I made these saves by running the game with this command every time:

`<path/to/woof.appimage> -iwad <path/to/doom2.wad>`

I'd really like to play with this launcher, but currently I can't because the saves of my WAD aren't showing up in my game when I run it with this launcher.

Can someone help with this issue? Your help would be really appreciated!

**Comment by Youda008** [comment]
Yes, i know about this. The way Doom Runner detects where save files are stored is currently broken. It will be fixed in the next major version.

**Comment by amethystgosling163** [comment]
When will it be out?

**Comment by Youda008** [comment]
I have no idea. Hopefully some time around Christmas.

**Comment by Youda008** [comment]
The fix is in master branch, so if you don't want to wait for all the features, testing and official release, you can still build it from sources.

**Comment by Youda008** [comment]
Please try it with the recent release 1.9.0, it should already work.

**Comment by Youda008** [comment]
@amethystgosling163 If you don't say it still doesn't work in 1 week, i will close this as completed.
